### PR TITLE
Create ncar.cc.xml

### PR DESCRIPTION
**Pull request by ivysrono**

### CODE REVIEW

File: src/chrome/content/rules/ncar.cc.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!--
+	MCB:
+		bbs.ncar.cc
+-->
+
+<ruleset name="ncar.cc">
+
+	<target host="bbs.ncar.cc" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/$" />
+	<exclusion pattern="^http://bbs\.ncar\.cc/.+\.(html|php)" />
```
Comment (fuglede) [inline src/chrome/content/rules/ncar.cc.xml]: From the looks of it, the tests are returning a false positive here, as they should have caught the unescaped dot in the exclusion pattern, cf. #7320.

Comment (ivysrono) [inline src/chrome/content/rules/ncar.cc.xml]: @fuglede How to fix it？

Comment (fuglede) [inline src/chrome/content/rules/ncar.cc.xml]: @ivysrono: That particular issue might mean that we will have to let some of these kinds of exclusions throught, but for now, just replace the unescaped dot by a suitable character class, such as `[\w/-]`, or whatever might be relevant for the pattern.

**Comment by J0WI** [comment]
There is also `beiyong.ncar.cc`

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**